charts/posthog/templates/clickhouse-operator/deployment.yaml

```diff
@@ -38,8 +47,8 @@ spec:
             name: etc-clickhouse-operator-usersd-files
       containers:
         - name: clickhouse-operator
-          image: altinity/clickhouse-operator:latest 
-          imagePullPolicy: IfNotPresent
+          image: altinity/clickhouse-operator:latest
+          imagePullPolicy: Always
```

Comment:
off topic

I'm guessing this is what you'd want to pulling `latest` all the time, but I guess this opens us up to unexpected updates now.

Reply:
I agree but running with `Always` is actually a k8s best practice

Reply:
Yup, is just the change coupled with `latest` being used tag that makes me think is potential for some deployments to suddenly pull an unwanted change. Just wanted to make sure its in mind.

But yeah, probably fine, non-blocking.

Reply:
👍 yup and I agree, this is anyway another change coming from upstream so we have little control over it